### ISSUE TITLE
Building in docker isn't working due to bundler version

### ISSUE THREAD

**Issue by AnnaDodson** [AnnaDodson/hacktoberfest-website#62]
**Is this issue a feature request or a bug report?**

- [ ] Feature
- [X] Bug

**What is the expected behaviour/what feature are you proposing?**

Doing the steps in the README to use docker to run the site locally should work without having to change any versions.

**What is the actual behaviour?**

Running `docker-compose up` doesn't build the site, this is the error in the console:

```
jekyll_1  | You must use Bundler 2 or greater with this lockfile.
hacktoberfest-website_jekyll_1 exited with code 20
``` 

**How can the bug be reproduced?**

Do not change any versions in the project and follow the steps in the README to run the docker build. The build fails and exits with the above error.

_*Note:* as a work around you can remove the bundler version in the Gemfile.lock to run the build through docker_

**Comment by fragmad** [comment]
I'll take a look at this.

**Comment by boardfish** [comment]
Noticed this myself - I think that particular Jekyll container is outdated compared to the current Dockerfile. I had a stab at trying to fix it myself.

**Comment by monotron** [comment]
Issue fixed by #65 - we should get this merged

**Comment by AnnaDodson** [comment]
Merged! Thanks for the comments confirming it was working for you both. I was having docker related problems so wasn't able to verify it.
